src/asmcnc/apps/drywall_cutter_app/screen_drywall_cutter.py

```diff
@@ -150,5 +190,39 @@ def save(self):
     def run(self):
         pass
 
+    def open_filechooser(self):
+        if not self.sm.has_screen('config_filechooser'):
+            self.sm.add_widget(screen_config_filechooser.ConfigFileChooser(name='config_filechooser',
+                                                                           screen_manager=self.sm,
+                                                                           localization=self.l,
+                                                                           callback=self.load_config))
+        self.sm.current = 'config_filechooser'
+
+    def load_config(self, config):
+        # type: (str) -> None
+        """
+        Used as the callback for the config filechooser screen.
+
+        :param config: The path to the config file, including extension.
+        """
+        self.dwt_config.load_config(config)
+
+        file_name_no_ext = config.split('/')[-1].split('.')[0]
+
+        # set the label on the screen to the name of the config file below
+
+        toolpath_offset = self.dwt_config.active_config.toolpath_offset
+        self.shape_selection.text = self.dwt_config.active_config.shape_type
+        self.select_shape()
```

Comment:
I think this will always set the rotation to horizontal as default, even if the config rotation is different?

Reply:
Rotation config did not exist when I created this pull request, but it automatically sets the rotation when the text inputs are filled in below this, which calls the x_input_change and y_input_change functions, which then sets the rotation based on the dimensions. I have now added a comment to explain this